test/processors/search_remover_test.rb

```diff
@@ -9,7 +9,7 @@ def setup
         </head>
         <body>
           <div id='global-header'>
-            <a href='#search'></a>
+            <a href='#search' class='search-toggle'></a>
```

Comment:
Do you think it's worth changing to a button to better reflect the soon-to-be-live change?

```suggestion
            <button href='#search' class='search-toggle'></button>
```

Reply:
Have done this (with the additional adjustment that if it's a button, it probably doesn't have a href).

Have also updated `CHANGELOG.md` with the change.

Thanks for approving ✅